openassessment/assessment/api/peer.py

```diff
@@ -45,7 +45,7 @@ def submitter_is_finished(submission_uuid, requirements):
         bool
 
     """
-    if requirements is None:
+    if not requirements:
```

Comment:
Nope, that should be okay. I just wanted to make sure that we wanted to behave this way when we get `{}` too.